### PR TITLE
Expose the visa booleans on the V3 api

### PR DESCRIPTION
**Pull request by davidgisbey**
### Context

We're about to start some work on Publish which allows providers to give information on whether they can sponsor skilled worker and/or student visas. Once we have captured this data, we will need to display it on find. This PR adds exposes the new visa booleans added in #1940 on the V3 provider serialiser

### Changes proposed in this pull request

- Exposes the `can_sponsor_skilled_worker_visa` and `can_sponsor_student_visa` booleans on the V3 provider serialiser
- Updates the `provider#show ` action to ensure they're passed to Find

### Trello card

https://trello.com/c/EukUPDxg/3442-ttapi-add-visa-to-the-v2-api

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
Will this be added for public v1 as well?

**Comment by davidgisbey** [comment]
Nope.jpg

This is only being applied for next cycles courses onwards.

**Comment by TobyRet** [comment]
I think the attr needs to be documented?
https://api.publish-teacher-training-courses.service.gov.uk/api-reference.html#schema-courseattributes

**Comment by davidgisbey** [comment]
@TobyRet so we're not exposing it on the public API for now. As far as i'm aware the v3 doesn't have docs right?

**Comment by TobyRet** [comment]
> @TobyRet so we're not exposing it on the public API for now. As far as i'm aware the v3 doesn't have docs right?

Woops I was thinking of Public / V1 - sorry!

File: app/controllers/api/v3/providers_controller.rb [hunk 1]
```diff
@@ -31,7 +31,8 @@ def show
                                        address3 address4 postcode region_code
                                        email website telephone train_with_us
                                        train_with_disability sites
-                                       accredited_bodies accredited_body?] }
+                                       accredited_bodies accredited_body?
+                                       can_sponsor_student_visa can_sponsor_skilled_worker_visa] }
```
Comment (TobyRet) [inline app/controllers/api/v3/providers_controller.rb:35]: new line req'd?

**Review by TobyRet — COMMENTED** [review]
One small formatting issue otherwise LGTM

**Review by TobyRet — APPROVED** [review]

**Pull request merged**